redbot/core/utils/antispam.py

```diff
@@ -8,15 +8,17 @@
 
 class AntiSpam:
     """
-    Custom class which is more flexible than using discord.py's
-    `commands.cooldown()`
+    An object that counts against temporary "intervals"
+    to reduce spam. These intervals can be set to incline
+    in duration for each interval that is stamped.
 
-    Can be intialized with a custom set of intervals
-    These should be provided as a list of tuples in the form
-    (timedelta, quantity)
-
-    Where quantity represents the maximum amount of times
-    something should be allowed in an interval.
+    Attributes
```

Comment:
These are not attributes, they're parameters:
```suggestion
    Parameters
```